src/pages/MenuPage.js

```diff
@@ -250,22 +251,25 @@ const mapDispatchToProps = dispatch => ({
 });
 
 const mapStateToProps = state => {
-  const { modules, user } = state;
-  const { currentUser } = user;
+  const { modules } = state;
+
   const {
     usingDashboard,
     usingDispensary,
     usingVaccines,
     usingCashRegister,
     usingModules,
   } = modules;
+
+  const isAdmin = selectCurrentUserIsAdmin(state);
```

Comment:
just changed to use a selector.. don't know how this got in here

![embares](https://user-images.githubusercontent.com/35858975/76271309-f6bb0400-62dc-11ea-8acb-e3696b647f93.gif)